src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -1912,8 +1912,8 @@ public interface I1
             var compilation1 = CreateStandardCompilation(source1, options: TestOptions.DebugDll,
                                                          parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
             Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
-            compilation1.VerifyDiagnostics(
-                // (4,9): error CS8052: Auto-implemented properties inside interfaces cannot have initializers.
+            compilation1.VerifyEmitDiagnostics(
```

Comment:
> It seems like we want to preserve the part where these errors are detected at binding.
 We do have a test like that.

---
In reply to: [121770162](https://github.com/dotnet/roslyn/pull/20165#discussion_r121770162) [](ancestors = 121770162,121768940)